Automattic/wp-calypso
# Stats: Updating CSS grid width percentage for 1/3 and 2/3 modules

Related to #87742 

## Proposed Changes

* updating changes introduced in #87743

## Testing Instructions



* open live branch in one tab
* open the live branch in another tab and apply `stats/flexible-grid` feature flag to the live branch
* verify that the page looks the same and modules are properly aligned on the Traffic page
* reduce the screen to below 1080px width and check that the modules take more space and the items are more readable
* verify that the `Posts` and `Referrers` modules take 60 and 40% instead of 66 and 33%

## Pre-merge Checklist
